js_basics/task4.js

```diff
@@ -0,0 +1,7 @@
+// Числа та булі
+
+let userAge = 7
+let ageOk = userAge >= 18
```

Comment:
**ageOk** name for variable does not represent the sense of this variable. Use **isAdult** name instead